SAM.Game/Manager.cs

```diff
@@ -459,12 +461,18 @@ private void GetAchievements()
 
                 if (textSearch != null)
                 {
-                    if (def.Name.IndexOf(textSearch, StringComparison.OrdinalIgnoreCase) < 0 ||
-                        def.Description.IndexOf(textSearch, StringComparison.OrdinalIgnoreCase) < 0)
+                    string[] searchTerms = textSearch.Split(new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries);
+
+                    bool nameMatches = searchTerms.All(term => def.Name.IndexOf(term, StringComparison.OrdinalIgnoreCase) >= 0);
+                    bool descriptionMatches = searchTerms.All(term => def.Description.IndexOf(term, StringComparison.OrdinalIgnoreCase) >= 0);
+
+                    if (!nameMatches && !descriptionMatches)
```

Comment:
Oh, I missed that it was added here, I've slightly reworked Search to be able to search with spaces, like 

![image](https://github.com/user-attachments/assets/8ca21049-1893-4b16-8c66-3349ff5476c8)

![image](https://github.com/user-attachments/assets/16c0761e-d992-41e1-852f-9e29a491a86f)

I'll revert this one if needed.